Core/Core/AppEnvironment/EnvironmentSettings/GetEnvironmentSettingsRequest.swift

```diff
@@ -17,7 +17,9 @@
 //
 
 public struct GetEnvironmentSettingsRequest: APIRequestable {
-    public typealias Response = [String: Bool]
+    public struct Response: Codable {
+        public let calendar_contexts_limit: Int?
```

Comment:
Can we say the it is confirmed the BE team will continue using this for future?
Or, perhaps, would be better to keep a logic to check for the Bool value as well? (The old way).

Reply:
I confirmed this with them and this will stay as-is. Thanks for bringing this up!